cookbooks/fb_helpers/libraries/node_methods.rb

```diff
@@ -110,12 +127,12 @@ def rhel10?
       self.rhel? && self['platform_version'].start_with?('10')
     end
 
-    def centos_min_version?(version)
-      self.centos? && self.el_min_version?(version)
+    def centos_min_version?(version, _full = false)
```

Comment:
What's the deal with not handling `full` here (and with oracle below)?  Are these platforms inherently never going to have non-integer versions?

Reply:
A typo, supposed to be passing in full here. Will fix, good catch!